iFixit/react-commerce
# Improve categories organization

Closes #123 and #161 

## Changelog

- Add alphabetical order to product list "choose a model" section
- Add manual override to alphabetical order via "SortPriority" field in Strapi
- Improve accessibility of "choose a model" section links

## QA

1. Visit the Vercel PR preview
2. Visit parts product list page
3. Verify that product lists under "choose a model" section are sorted alphabetically
4. Login into Strapi (https://improve-categories-organization.govinor.com/admin/ with `strapi@ifixit.com` and `Password1`) 
5. Select a product list and set the sort priority field to a number higher than 0 (e.g. 10)
6. Refresh the parts product list page and verify that the edited product list is shown at the top in the "choose a model section"
7. Verify that an outline border is displayed when a link of the "choose a model" section is in focus (e.g. navigate using tab)
